data/json/statistics.json

```diff
@@ -337,5 +337,117 @@
     "stat_type": "last_value",
     "event_transformation": "avatar_enters_oter_type",
     "field": "oter_type_id"
+  },
+  {
+    "id": "statistic_cuts_tree",
+    "type": "event_transformation",
+    "event_type": "cuts_tree",
+    "value_constraints": { "character": { "equals_statistic": "avatar_id" } },
+    "drop_fields": [ "character" ]
+  },
```

Comment:
On the subject of naming conventions: I'd like to have `avatar` in the name of any `event_transformation` where the events have been constrained to just-the-avatar (as they have in this case).  I would have called this one `avatar_cuts_tree`.